src/dappsExplorer/DAppsExplorerScreenSearchFilter.test.tsx

```diff
@@ -655,5 +658,30 @@ describe(DAppsExplorerScreenSearchFilter, () => {
       expect(within(allDappsSection).queryByText(dappsList[1].name)).toBeFalsy()
       expect(within(allDappsSection).queryByText(dappsList[1].description)).toBeFalsy()
     })
+
+    it('shows QR button when experiment enabled', () => {
+      mocked(getExperimentParams).mockReturnValue({
+        showQrScanner: true,
+      })
+
+      const { queryByTestId } = render(
+        <Provider store={defaultStore}>
+          <DAppsExplorerScreenSearchFilter />
+        </Provider>
+      )
+      expect(queryByTestId('DAppsExplorerScreen/QRScanButton')).toBeTruthy()
```

Comment:
nit: we should use the `getBy*` methods rather than `queryBy*` for positive assertions because the error messages are more helpful if the test fails